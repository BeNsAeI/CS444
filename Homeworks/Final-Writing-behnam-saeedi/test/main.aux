\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0 << /S /D >> }
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{4}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Linux}{4}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Process}{4}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1}Process}{4}{subsubsection.2.1.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces The view of "top" command in Linux shell}}{5}{figure.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces The view of "ps fax" command in shell}}{6}{figure.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.2}Thread}{6}{subsubsection.2.1.2}}
\citation{cscedu}
\citation{cfsmemo}
\citation{cfshw}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.3}CPU Scheduling}{7}{subsubsection.2.1.3}}
\citation{1}
\citation{2}
\citation{3}
\citation{4}
\citation{5}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}I/O}{8}{subsection.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1}Data Structure}{8}{subsubsection.2.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.2}Algorithms}{8}{subsubsection.2.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.3}I/O scheduling}{8}{subsubsection.2.2.3}}
\citation{6}
\citation{2-1}
\citation{2-1}
\citation{2-1}
\citation{2-2}
\citation{2-1}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.4}Cryptography}{9}{subsubsection.2.2.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}Memory Management}{9}{subsection.2.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.3.1}Memory Management}{9}{subsubsection.2.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.3.2}Services}{9}{subsubsection.2.3.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.2.1}VM}{9}{paragraph.2.3.2.1}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.2.2}Address Space Protection}{9}{paragraph.2.3.2.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.2.3}Memory Mapping}{9}{paragraph.2.3.2.3}}
\citation{2-1}
\citation{2-1}
\citation{2-1}
\citation{2-3}
\citation{2-1}
\citation{2-1}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.3.3}Pages}{10}{subsubsection.2.3.3}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.3.1}Efficiency and Demand Paging}{10}{paragraph.2.3.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.3.4}Features}{10}{subsubsection.2.3.4}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.1}Cache}{10}{paragraph.2.3.4.1}}
\newlabel{BC}{{2.3.4.1}{10}{Cache\relax }{paragraph.2.3.4.1}{}}
\newlabel{HWC}{{2.3.4.1}{10}{Cache\relax }{paragraph.2.3.4.1}{}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.2}Swapping}{10}{paragraph.2.3.4.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.3}Threads}{10}{paragraph.2.3.4.3}}
\newlabel{SVM}{{2.3.4.4}{10}{Shared Virtual Memory\relax }{paragraph.2.3.4.4}{}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.4}Shared Virtual Memory}{10}{paragraph.2.3.4.4}}
\citation{2-1}
\citation{2-1}
\citation{msdnprocess}
\citation{msdnprocess}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.5}Access Control}{11}{paragraph.2.3.4.5}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Meaning of bit fields and OS control on access.\cite  {2-1}}}{11}{table.1}}
\newlabel{bitfields}{{1}{11}{Meaning of bit fields and OS control on access.\cite {2-1}\relax }{table.1}{}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {2.3.4.6}Paging tables}{11}{paragraph.2.3.4.6}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Windows}{11}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Process}{11}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.1.1}Process}{11}{subsubsection.3.1.1}}
\citation{msdnprocess}
\citation{msdnprocess}
\citation{msdnprocess}
\citation{windowscpuscheduling}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.1.1}Command: Tasklist}{12}{paragraph.3.1.1.1}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.1.2}Close Program}{12}{paragraph.3.1.1.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.1.3}Task Manager}{12}{paragraph.3.1.1.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.1.2}Thread}{12}{subsubsection.3.1.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.1.3}Fiber}{12}{subsubsection.3.1.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.1.4}CPU Scheduling}{12}{subsubsection.3.1.4}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.4.1}Priorities}{12}{paragraph.3.1.4.1}}
\citation{winconsw}
\citation{windowscpuscheduling}
\citation{7}
\citation{8}
\citation{9}
\citation{10}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.4.2}Context Switching}{13}{paragraph.3.1.4.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {3.1.4.3}Algorithm}{13}{paragraph.3.1.4.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}I/O}{13}{subsection.3.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.2.1}Data Structure}{13}{subsubsection.3.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.2.2}I/O scheduling}{13}{subsubsection.3.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.2.3}Cryptography}{14}{subsubsection.3.2.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.3}Memory Management}{14}{subsection.3.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.1}Memory Management}{14}{subsubsection.3.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.2}Size}{14}{subsubsection.3.3.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.3}Service}{14}{subsubsection.3.3.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.4}Page}{15}{subsubsection.3.3.4}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.5}Feature}{15}{subsubsection.3.3.5}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {3.3.6}Comparing to Linux}{15}{subsubsection.3.3.6}}
\@writefile{toc}{\contentsline {section}{\numberline {4}FreeBSD}{15}{section.4}}
\citation{freebsdPD}
\citation{freebsdalook}
\citation{freebsdpdf}
\citation{11}
\citation{freebsdpdf}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Process}{16}{subsection.4.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1}Process}{16}{subsubsection.4.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.2}Thread}{16}{subsubsection.4.1.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.3}CPU Scheduling}{16}{subsubsection.4.1.3}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {4.1.3.1}4.4 BSD scheduler}{16}{paragraph.4.1.3.1}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {4.1.3.2}ULE Scheduler}{16}{paragraph.4.1.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}I/O}{16}{subsection.4.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.2.1}Data Structure}{16}{subsubsection.4.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.2.2}I/O scheduling}{16}{subsubsection.4.2.2}}
\citation{12}
\citation{13}
\citation{11}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.2.3}Cryptography}{17}{subsubsection.4.2.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.2.4}Kernel System Queue Synopsis}{17}{subsubsection.4.2.4}}
\citation{2-4}
\citation{2-5}
\citation{2-4}
\citation{2-4}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.3}Memory Management}{19}{subsection.4.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.3.1}Memory Management}{19}{subsubsection.4.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.3.2}Pages}{19}{subsubsection.4.3.2}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {4.3.2.1}States of Memory}{19}{paragraph.4.3.2.1}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {4.3.2.2}Swapping}{19}{paragraph.4.3.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.3.3}Services and Features}{19}{subsubsection.4.3.3}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {4.3.3.1}Generic VM Objects}{20}{paragraph.4.3.3.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.3.4}FreeBSD and Linux Comparison}{20}{subsubsection.4.3.4}}
\@writefile{toc}{\contentsline {section}{\numberline {5}conclusion}{20}{section.5}}
\bibstyle{IEEEtran}
\bibdata{sources}
\gdef\minted@oldcachelist{,
  default-pyg-prefix.pygstyle,
  default.pygstyle,
  576F8C5E472CA804F36BF4DD36996B9A5093E84BEA004B2911D4DA186861F6DB.pygtex,
  0A46636B0B2011FEE9100DD10389D71A5093E84BEA004B2911D4DA186861F6DB.pygtex,
  77B48E86AA42AB4ECD97AB9CB1433CF65093E84BEA004B2911D4DA186861F6DB.pygtex,
  0295453ADF5E8DDB9A0766BEBCA1EEFB5093E84BEA004B2911D4DA186861F6DB.pygtex,
  CB42047863FF678A703C549029628A5E5093E84BEA004B2911D4DA186861F6DB.pygtex,
  FE383A005C1097285B99F58419692D8F042D2615758B6C3F03C186B02CFC62B2.pygtex}
