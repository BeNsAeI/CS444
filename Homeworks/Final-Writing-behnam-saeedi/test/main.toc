\select@language {english}
\contentsline {section}{\numberline {1}Introduction}{4}{section.1}
\contentsline {section}{\numberline {2}Linux}{4}{section.2}
\contentsline {subsection}{\numberline {2.1}Process}{4}{subsection.2.1}
\contentsline {subsubsection}{\numberline {2.1.1}Process}{4}{subsubsection.2.1.1}
\contentsline {subsubsection}{\numberline {2.1.2}Thread}{6}{subsubsection.2.1.2}
\contentsline {subsubsection}{\numberline {2.1.3}CPU Scheduling}{7}{subsubsection.2.1.3}
\contentsline {subsection}{\numberline {2.2}I/O}{8}{subsection.2.2}
\contentsline {subsubsection}{\numberline {2.2.1}Data Structure}{8}{subsubsection.2.2.1}
\contentsline {subsubsection}{\numberline {2.2.2}Algorithms}{8}{subsubsection.2.2.2}
\contentsline {subsubsection}{\numberline {2.2.3}I/O scheduling}{8}{subsubsection.2.2.3}
\contentsline {subsubsection}{\numberline {2.2.4}Cryptography}{9}{subsubsection.2.2.4}
\contentsline {subsection}{\numberline {2.3}Memory Management}{9}{subsection.2.3}
\contentsline {subsubsection}{\numberline {2.3.1}Memory Management}{9}{subsubsection.2.3.1}
\contentsline {subsubsection}{\numberline {2.3.2}Services}{9}{subsubsection.2.3.2}
\contentsline {paragraph}{\numberline {2.3.2.1}VM}{9}{paragraph.2.3.2.1}
\contentsline {paragraph}{\numberline {2.3.2.2}Address Space Protection}{9}{paragraph.2.3.2.2}
\contentsline {paragraph}{\numberline {2.3.2.3}Memory Mapping}{9}{paragraph.2.3.2.3}
\contentsline {subsubsection}{\numberline {2.3.3}Pages}{10}{subsubsection.2.3.3}
\contentsline {paragraph}{\numberline {2.3.3.1}Efficiency and Demand Paging}{10}{paragraph.2.3.3.1}
\contentsline {subsubsection}{\numberline {2.3.4}Features}{10}{subsubsection.2.3.4}
\contentsline {paragraph}{\numberline {2.3.4.1}Cache}{10}{paragraph.2.3.4.1}
\contentsline {paragraph}{\numberline {2.3.4.2}Swapping}{10}{paragraph.2.3.4.2}
\contentsline {paragraph}{\numberline {2.3.4.3}Threads}{10}{paragraph.2.3.4.3}
\contentsline {paragraph}{\numberline {2.3.4.4}Shared Virtual Memory}{10}{paragraph.2.3.4.4}
\contentsline {paragraph}{\numberline {2.3.4.5}Access Control}{11}{paragraph.2.3.4.5}
\contentsline {paragraph}{\numberline {2.3.4.6}Paging tables}{11}{paragraph.2.3.4.6}
\contentsline {section}{\numberline {3}Windows}{11}{section.3}
\contentsline {subsection}{\numberline {3.1}Process}{11}{subsection.3.1}
\contentsline {subsubsection}{\numberline {3.1.1}Process}{11}{subsubsection.3.1.1}
\contentsline {paragraph}{\numberline {3.1.1.1}Command: Tasklist}{12}{paragraph.3.1.1.1}
\contentsline {paragraph}{\numberline {3.1.1.2}Close Program}{12}{paragraph.3.1.1.2}
\contentsline {paragraph}{\numberline {3.1.1.3}Task Manager}{12}{paragraph.3.1.1.3}
\contentsline {subsubsection}{\numberline {3.1.2}Thread}{12}{subsubsection.3.1.2}
\contentsline {subsubsection}{\numberline {3.1.3}Fiber}{12}{subsubsection.3.1.3}
\contentsline {subsubsection}{\numberline {3.1.4}CPU Scheduling}{12}{subsubsection.3.1.4}
\contentsline {paragraph}{\numberline {3.1.4.1}Priorities}{12}{paragraph.3.1.4.1}
\contentsline {paragraph}{\numberline {3.1.4.2}Context Switching}{13}{paragraph.3.1.4.2}
\contentsline {paragraph}{\numberline {3.1.4.3}Algorithm}{13}{paragraph.3.1.4.3}
\contentsline {subsection}{\numberline {3.2}I/O}{13}{subsection.3.2}
\contentsline {subsubsection}{\numberline {3.2.1}Data Structure}{13}{subsubsection.3.2.1}
\contentsline {subsubsection}{\numberline {3.2.2}I/O scheduling}{13}{subsubsection.3.2.2}
\contentsline {subsubsection}{\numberline {3.2.3}Cryptography}{14}{subsubsection.3.2.3}
\contentsline {subsection}{\numberline {3.3}Memory Management}{14}{subsection.3.3}
\contentsline {subsubsection}{\numberline {3.3.1}Memory Management}{14}{subsubsection.3.3.1}
\contentsline {subsubsection}{\numberline {3.3.2}Size}{14}{subsubsection.3.3.2}
\contentsline {subsubsection}{\numberline {3.3.3}Service}{14}{subsubsection.3.3.3}
\contentsline {subsubsection}{\numberline {3.3.4}Page}{15}{subsubsection.3.3.4}
\contentsline {subsubsection}{\numberline {3.3.5}Feature}{15}{subsubsection.3.3.5}
\contentsline {subsubsection}{\numberline {3.3.6}Comparing to Linux}{15}{subsubsection.3.3.6}
\contentsline {section}{\numberline {4}FreeBSD}{15}{section.4}
\contentsline {subsection}{\numberline {4.1}Process}{16}{subsection.4.1}
\contentsline {subsubsection}{\numberline {4.1.1}Process}{16}{subsubsection.4.1.1}
\contentsline {subsubsection}{\numberline {4.1.2}Thread}{16}{subsubsection.4.1.2}
\contentsline {subsubsection}{\numberline {4.1.3}CPU Scheduling}{16}{subsubsection.4.1.3}
\contentsline {paragraph}{\numberline {4.1.3.1}4.4 BSD scheduler}{16}{paragraph.4.1.3.1}
\contentsline {paragraph}{\numberline {4.1.3.2}ULE Scheduler}{16}{paragraph.4.1.3.2}
\contentsline {subsection}{\numberline {4.2}I/O}{16}{subsection.4.2}
\contentsline {subsubsection}{\numberline {4.2.1}Data Structure}{16}{subsubsection.4.2.1}
\contentsline {subsubsection}{\numberline {4.2.2}I/O scheduling}{16}{subsubsection.4.2.2}
\contentsline {subsubsection}{\numberline {4.2.3}Cryptography}{17}{subsubsection.4.2.3}
\contentsline {subsubsection}{\numberline {4.2.4}Kernel System Queue Synopsis}{17}{subsubsection.4.2.4}
\contentsline {subsection}{\numberline {4.3}Memory Management}{19}{subsection.4.3}
\contentsline {subsubsection}{\numberline {4.3.1}Memory Management}{19}{subsubsection.4.3.1}
\contentsline {subsubsection}{\numberline {4.3.2}Pages}{19}{subsubsection.4.3.2}
\contentsline {paragraph}{\numberline {4.3.2.1}States of Memory}{19}{paragraph.4.3.2.1}
\contentsline {paragraph}{\numberline {4.3.2.2}Swapping}{19}{paragraph.4.3.2.2}
\contentsline {subsubsection}{\numberline {4.3.3}Services and Features}{19}{subsubsection.4.3.3}
\contentsline {paragraph}{\numberline {4.3.3.1}Generic VM Objects}{20}{paragraph.4.3.3.1}
\contentsline {subsubsection}{\numberline {4.3.4}FreeBSD and Linux Comparison}{20}{subsubsection.4.3.4}
\contentsline {section}{\numberline {5}conclusion}{20}{section.5}
