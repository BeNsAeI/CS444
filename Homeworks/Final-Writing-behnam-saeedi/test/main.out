\BOOKMARK [1][-]{section.1}{Introduction}{}
\BOOKMARK [1][-]{section.2}{Linux}{}
\BOOKMARK [2][-]{subsection.2.1}{Process}{section.2}
\BOOKMARK [3][-]{subsubsection.2.1.1}{Process}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.2}{Thread}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.3}{CPU Scheduling}{subsection.2.1}
\BOOKMARK [2][-]{subsection.2.2}{I/O}{section.2}
\BOOKMARK [3][-]{subsubsection.2.2.1}{Data Structure}{subsection.2.2}
\BOOKMARK [3][-]{subsubsection.2.2.2}{Algorithms}{subsection.2.2}
\BOOKMARK [3][-]{subsubsection.2.2.3}{I/O scheduling}{subsection.2.2}
\BOOKMARK [3][-]{subsubsection.2.2.4}{Cryptography}{subsection.2.2}
\BOOKMARK [2][-]{subsection.2.3}{Memory Management}{section.2}
\BOOKMARK [3][-]{subsubsection.2.3.1}{Memory Management}{subsection.2.3}
\BOOKMARK [3][-]{subsubsection.2.3.2}{Services}{subsection.2.3}
\BOOKMARK [3][-]{subsubsection.2.3.3}{Pages}{subsection.2.3}
\BOOKMARK [3][-]{subsubsection.2.3.4}{Features}{subsection.2.3}
\BOOKMARK [1][-]{section.3}{Windows}{}
\BOOKMARK [2][-]{subsection.3.1}{Process}{section.3}
\BOOKMARK [3][-]{subsubsection.3.1.1}{Process}{subsection.3.1}
\BOOKMARK [3][-]{subsubsection.3.1.2}{Thread}{subsection.3.1}
\BOOKMARK [3][-]{subsubsection.3.1.3}{Fiber}{subsection.3.1}
\BOOKMARK [3][-]{subsubsection.3.1.4}{CPU Scheduling}{subsection.3.1}
\BOOKMARK [2][-]{subsection.3.2}{I/O}{section.3}
\BOOKMARK [3][-]{subsubsection.3.2.1}{Data Structure}{subsection.3.2}
\BOOKMARK [3][-]{subsubsection.3.2.2}{I/O scheduling}{subsection.3.2}
\BOOKMARK [3][-]{subsubsection.3.2.3}{Cryptography}{subsection.3.2}
\BOOKMARK [2][-]{subsection.3.3}{Memory Management}{section.3}
\BOOKMARK [3][-]{subsubsection.3.3.1}{Memory Management}{subsection.3.3}
\BOOKMARK [3][-]{subsubsection.3.3.2}{Size}{subsection.3.3}
\BOOKMARK [3][-]{subsubsection.3.3.3}{Service}{subsection.3.3}
\BOOKMARK [3][-]{subsubsection.3.3.4}{Page}{subsection.3.3}
\BOOKMARK [3][-]{subsubsection.3.3.5}{Feature}{subsection.3.3}
\BOOKMARK [3][-]{subsubsection.3.3.6}{Comparing to Linux}{subsection.3.3}
\BOOKMARK [1][-]{section.4}{FreeBSD}{}
\BOOKMARK [2][-]{subsection.4.1}{Process}{section.4}
\BOOKMARK [3][-]{subsubsection.4.1.1}{Process}{subsection.4.1}
\BOOKMARK [3][-]{subsubsection.4.1.2}{Thread}{subsection.4.1}
\BOOKMARK [3][-]{subsubsection.4.1.3}{CPU Scheduling}{subsection.4.1}
\BOOKMARK [2][-]{subsection.4.2}{I/O}{section.4}
\BOOKMARK [3][-]{subsubsection.4.2.1}{Data Structure}{subsection.4.2}
\BOOKMARK [3][-]{subsubsection.4.2.2}{I/O scheduling}{subsection.4.2}
\BOOKMARK [3][-]{subsubsection.4.2.3}{Cryptography}{subsection.4.2}
\BOOKMARK [3][-]{subsubsection.4.2.4}{Kernel System Queue Synopsis}{subsection.4.2}
\BOOKMARK [2][-]{subsection.4.3}{Memory Management}{section.4}
\BOOKMARK [3][-]{subsubsection.4.3.1}{Memory Management}{subsection.4.3}
\BOOKMARK [3][-]{subsubsection.4.3.2}{Pages}{subsection.4.3}
\BOOKMARK [3][-]{subsubsection.4.3.3}{Services and Features}{subsection.4.3}
\BOOKMARK [3][-]{subsubsection.4.3.4}{FreeBSD and Linux Comparison}{subsection.4.3}
\BOOKMARK [1][-]{section.5}{conclusion}{}
